src/helpers/set-commit-status.ts

```diff
@@ -23,17 +24,33 @@ export class SetCommitStatus extends HelperInputs {
   state = '';
   description?: string;
   target_url?: string;
+  skip_if_already_set?: string;
 }
 
-export const setCommitStatus = async ({ sha, context, state, description, target_url }: SetCommitStatus) => {
-  await map(context.split('\n').filter(Boolean), context =>
+export const setCommitStatus = async ({ sha, context, state, description, target_url, skip_if_already_set }: SetCommitStatus) => {
+  await map(context.split('\n').filter(Boolean), async context => {
+    if (skip_if_already_set === 'true') {
```

Comment:
I think we can simplify to this since you're either omitting this or passing `true`
```suggestion
    if (skip_if_already_set) {
```

Reply:
technically yeah, but allowing `'false'` to evaluate to `true` here seems kinda weird